app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,13 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
 
-  def show; end
+    posts_list = Post.all.order(created_at: :desc)
+    @posts = posts_list.select { |i| i['id'] == @user[:id] }
```

Comment:
- [ ] <b>REQUIRED:</b> As the prvious review requested, Kindly use `recent_post` method you created in user model. 

```ruby
@post = @user.recent_post

# Or, if you want to get all posts.

@post = @user.posts